docs/enterprise-setup/implementation-guide.md

```diff
@@ -288,6 +205,18 @@ To deploy Self-Managed Enterprise without SSO, exclude the entire `auth:` sectio
 ```yml
 global:
   edition: enterprise
+
+  # This must be set to the public facing URL of your Airbyte instance.
+  airbyteUrl: https://airbyte.company.example
+
+  # Optional configuration of an OIDC provider
+  auth:
+    identityProvider:
+      type: oidc
+      domain: company.example
+      app-name: airbyte
+      client-id: e83bbc57-1991-417f-8203-3affb47636cf
+      client-secret: $OKTA_CLIENT_SECRET
```

Comment:
@perangel I don't think I got this right. I could not find what this was supposed to be set to in the PR.